docs/src/main/sphinx/security/oauth2.rst

```diff
@@ -201,13 +235,36 @@ The following configuration properties are available:
      - Base64-encoded secret key used to encrypt the generated token.
        By default it's generated during startup.
 
+.. _trino-oauth2-troubleshooting:
 
 Troubleshooting
 ---------------
 
-If you need to debug issues with Trino OAuth 2.0 configuration you can change
+If you need to debug issues with Trino OAuth 2.0 configuration, you can change
 the :ref:`log level <log-levels>` for the OAuth 2.0 authenticator:
 
 .. code-block:: none
 
     io.trino.server.security.oauth2=DEBUG
+
+If you need to debug issues with Trino OAuth 2.0 Web UI, you can set the
+following configuration property:
+
+.. code-block:: none
+
+    io.trino.server.ui.OAuth2WebUiAuthenticationFilter=DEBUG
+
+This assumes the OAuth 2.0 authentication for the Web UI is enabled as described
+in :ref:`trino-server-configuration-oauth2`.
+
+The logged debug error is ``Tokens refresh challenge has failed``.
+
+.. warning::
+
+    If a refresh token lapses, a user session is interrupted and the user has to
+    reauthenticate by logging in again. Be sure the
+    ``http-server.authentication.oauth2.refresh-tokens.issued-token.timeout``
+    value is set to less than or equal to the duration of the refresh token
```

Comment:
You must set the ``blahblahblah`` value to less than...

Reply:
Went with that general construction but retained "Be sure" rather than "You must" (or "ensure"). There is a default that may or may not work depending on IdP  token TTLs.